### ISSUE TITLE
Map change is broken.

### ISSUE THREAD

**Issue by XGNXLency** [JetBoom/zombiesurvival#209]
For some reason, the server won't change maps after the round is finished even though all of the map files are in the maps folder, as well as in the maps folder in the base Garry's Mod maps folder AND in several addons folders in our addons. This has been an issue since I upgraded from the 2012 ZS. I'm currently running on the latest version. When I installed a MapVote addon, the menu would appear blank. Any fixes?

**Comment by arthurgeron** [comment]
This is probably because the "votemap" addon you use used a older hook, that's not an issue with the gamemode itself.

**Comment by arthurgeron** [comment]
Ask help on forums, with the creator of the addon or for some LUA developer that you know.

**Comment by XGNXLency** [comment]
I don't think you understand what I'm saying. The map change system is BROKEN. PERIOD. I've tried it on COMPLETE DEFAULT without ANY Map Vote addons and it would NOT change the map. It would just sit on "Loading. . ." Also, what forums? You mean the FacePunch forums that were literally just shut down a few months ago or the new crappy refugee forum for the remnants of FacePunch?

**Comment by arthurgeron** [comment]
Delete everything from your server and start it from scratch, extract the gamemode in the gamemode folder and it will work.

If zombies win too fast it'll start a second round without changing maps.

**Comment by arthurgeron** [comment]
It's not broken, I've been using this for a long time without any issues. You clearly understand 0 of programming and at the same time is sure of everything and demanding a fix? Maybe you should hire a dev for your server, you won't get anyone to fix your shit here.

**Comment by XGNXLency** [comment]
I understand a lot about programming. There are NO lua errors. I've talked with MANY developers and all of them say it's nothing related to anything I've modified, added, or removed. You're probably right about not getting anyone to fix it here. Everyone here is just as cancerous as the other ZS server devs I've spoken with.

**Comment by arthurgeron** [comment]
Great, then find a fix for it yourself or ask your lots of friends. Bye

**Comment by arthurgeron** [comment]
oh and for your question"Any fixes?" the answer is no

**Comment by XGNXLency** [comment]
Could have just lead with that smartass. Secondly, you must be retarded and/or illiterate because you literally just told me to do something I already said I did. 10/10 support from this jackass here.

**Comment by arthurgeron** [comment]
So let me get this straight, you come here to report an "issue", and with a " Any fixes?" attitude.
Rages as soon as you see a comment you disagree with. 
Talks about how hard it's to get support currently (like that's our concern).
Tries to impose yourself as a "coder" with a lot of dev friends. 
And in the end still expects for someone here to fix your stuff for you?
How entitled! You literally never used GitHub before and just created this account to demand for some magic help to fix your problems, what a experienced coder you are!
Hope your extended vocabulary and politeness gets you farthest than it looks.

All that eerie attitude earned me some laughs.

Have fun

**Comment by JetBoom** [comment]
duhweuaehuwahueawaaaa
